### DISCUSSION TITLE
setCurrentValue is not updating sceen with new value

### DISCUSSION THREAD

**Discussion by evanderkuy** [davetcc/tcMenu#469] [General]
I have created a tcMenu using an encoder and a OLED display, This working perfectly fine now on the device. I am also using BLE to make changes to the same settings on the device. When the user is using BLE, I want the menu item also to show the new value.

I have tried to use the setCurrentValue() for that menu item, and also tried changeOccurred(), menuMgr.notifyStructureChanged() and menuMgr.valueChanged() and in combinations but it won't show the new value.

Is it possible to change current values in menu items in this way without having to deal with "remote control"?

**Comment by tomelgato** [comment]
Normally changes via xy.setCurrentValue() are displayed without further code. What type of menuItem?

**Comment by evanderkuy** [comment]
Hi Dave,

It is working fine now, don't know what went wrong. Thx!
